Index: script.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/script.sh b/bash-script/script.sh
rename from script.sh
rename to bash-script/script.sh
--- a/script.sh	(revision 310e8ceb01294643358e00732d3f608cb6785754)
+++ b/bash-script/script.sh	(date 1658459857535)
@@ -13,4 +13,4 @@
 M2_HOME=/opt/maven/bin
 JAVA_HOME=$(find /usr/lib -name java-11*)
 echo PATH=$PATH:$HOME/bin:$JAVA_HOME:$M2:$M2_HOME >> ~/.bashrc
-/opt/tomcat/bin/startup.sh
+/opt/tomcat/bin/startup.sh
\ No newline at end of file
